bootstrap/bootstrap.go

```diff
@@ -85,6 +88,42 @@ func (b *Bootstrap) Run(ctx context.Context) (exitCode int) {
 	defer stopper()
 	defer func() { span.FinishWithError(err) }()
 
+	if experiments.IsEnabled("job-api") {
+		if runtime.GOOS == "windows" {
+			b.shell.Errorf("The job API is not currently supported on Windows. It's on its way though, promise!")
+			return 1
+		}
+
+		jobAPISocketPath := path.Join(os.TempDir(), fmt.Sprintf("buildkite-agent-%d.sock", os.Getpid()))
+		srv, token, err := jobapi.NewServer(jobAPISocketPath, b.shell.Env)
+		if err != nil {
+			b.shell.Errorf("Error creating job API server: %v", err)
+			return 1
+		}
+
+		b.shell.Env.Set("BUILDKITE_AGENT_JOB_API_SOCKET", jobAPISocketPath)
+		b.shell.Env.Set("BUILDKITE_AGENT_JOB_API_TOKEN", token)
+
+		err = srv.Start()
```

Comment:
```suggestion
		if err := srv.Start(); err != nil {
```

I prefer this kind of thing; for one reason, the scope of `err` is limited to the `if`, which it makes it clear there's no further processing of the value for `err`.